test/gtest/mhp/CMakeLists.txt

```diff
@@ -63,6 +64,7 @@ foreach(test-exec IN ITEMS mhp-tests mhp-tests-3 mhp-quick-test)
     cxxopts
     DR::mpi
   )
+
```

Comment:
Next time please avoid making empty changes to CMakeFiles (like empty spaces or lines). They make review harder.
If they are made by your IDE try to find option to disable them or clear them before making PR.